core/src/test/resources/example.json

```diff
@@ -0,0 +1,76 @@
+{
+  "dataSources": {
+    "wikipedia": {
+      "spec": {
+        "dataSchema": {
+          "dataSource": "wikipedia",
+          "parser": {
+            "type": "string",
+            "parseSpec": {
+              "format": "json",
+              "timestampSpec": {
+                "column": "timestamp",
+                "format": "auto"
+              },
+              "dimensionsSpec": {
+                "dimensions": [
+                  "page",
+                  "language",
+                  "user",
+                  "unpatrolled",
+                  "newPage",
+                  "robot",
+                  "anonymous",
+                  "namespace",
+                  "continent",
+                  "country",
+                  "region",
+                  "city"
+                ],
+                "dimensionExclusions": [],
+                "spatialDimensions": []
+              }
+            }
+          },
+          "metricsSpec": [
+            {
+              "type": "count",
+              "name": "count"
+            },
+            {
+              "type": "doubleSum",
+              "name": "added",
+              "fieldName": "added"
+            },
+            {
+              "type": "doubleSum",
+              "name": "deleted",
+              "fieldName": "deleted"
+            },
+            {
+              "type": "doubleSum",
+              "name": "delta",
+              "fieldName": "delta"
+            }
+          ],
+          "granularitySpec": {
+            "type": "uniform",
+            "segmentGranularity": "DAY",
+            "queryGranularity": "NONE"
+          }
+        },
+        "tuningConfig": {
```

Comment:
They do although currently tranquility doesn't support all of them. see also #116